modules/reindex/build.gradle

```diff
@@ -146,7 +148,7 @@ if (Os.isFamily(Os.FAMILY_WINDOWS)) {
       env 'JAVA_HOME', jdks.legacy.javaHomePath
       args 'oldes.OldElasticsearch',
         baseDir,
-        unzip.temporaryDir,
+        unzip.get().temporaryDir,
```

Comment:
Doesn't this effectively nullify any benefit of lazily creating the unzip task? Is there a reason we can't lazily create the fixture task?

Reply:
yeah this just kicks the can down to the fixture task creation. Changing these fixtures to use task avoidance had some side effects in an earlier trial I want to look in an dedicated effort. Again this PR is basically of just porting the low hanging fruits